templates/packages/order_edit.html

```diff
@@ -0,0 +1,217 @@
+{%extends 'base.html' %}
+
+{% block title%}
+     הזמנה {{order.collection_date.date}}
+{%endblock%}
+
+{% block content %}
+    <div class="row justify-content-center mb-3">
+        <div class="col-auto order-2">
+            <a class="btn btn-outline-dark" data-bs-toggle="modal" data-bs-target="#package-modal">חבילה חדשה +</a>

```

Comment:
You might want to clear the modal here
![dick modal](https://i.imgur.com/lF1Xg9j.png)

Reply:
I haven't decided if it will be done on the submit or new... Because accidental closure of the modal is unlikely because of it's static configuration, but the user still might wanna close it temporarily and get back to the filling out. What do you think?